packages/actor-expression-evaluator-factory-base/lib/ActorExpressionEvaluatorFactoryBase.ts

```diff
@@ -0,0 +1,131 @@
+import type {
```

Comment:
Aha, I was thinking like an hour about it hahah, yes! I think it used to be because you could use the function helpers when using the package. But that might not be the case anymore. Since functions is it's own bus already :)

Reply:
It's not really a base actor I think. It's a true implementation of the actor. (It's used by the engines) I think we should look for a more suitable name. Can I call it `ExpressionEvaluatorFactorySparqlee`?

Reply:
But would it hurt to move it to the bus package, just to avoid the need for another package that most actors on that bus will depend on anyway?

Regarding the name, I'd avoid "SPARQL" in the name, as we aim to be more generic than SPARQL.

Reply:
Since the actor is an actual actor. I will rename it to `actor-expression-evaluator-factory-default`.